app.rb

```diff
@@ -43,7 +43,20 @@ def add_music_album
   end
 
   def add_movie
-    puts 'Book Added Succefully!!'
+    puts 'Author'
+    author = gets.chomp
+    puts 'Genre'
+    genre = gets.chomp
+    puts 'Label'
+    label = gets.chomp
+    puts 'Source'
+    source = gets.chomp
+    movie = Movie.new('2022/2/2', silent: true)
+    movie.author = author
+    movie.genre = genre
+    movie.label = label
+    movie.source = source
+    movie.add_new_movie(movie)
```

Comment:
- [ ] There is no specification about implementing the `add_movie` method for this task, and since we decided ahead on how we should implement those methods in our discussion as a team, so to avoid any branch conflicts as well, kindly leave this implementation for the coming task.